.github/CODEOWNERS

```diff
@@ -611,6 +611,8 @@ peps/pep-0730.rst  @ned-deily
 peps/pep-0731.rst  @gvanrossum @encukou @vstinner @zooba @iritkatriel
 peps/pep-0732.rst  @Mariatta
 peps/pep-0733.rst  @encukou @vstinner @zooba @iritkatriel
+peps/pep-0734.rst  @vstinner
```

Comment:
735 not 734 :)

734 is https://github.com/python/peps/pull/3523

Reply:
Meanwhile I should have hit the right empty PEP slot.